packages/cli/src/api/pseudoLocalize.test.js

```diff
@@ -35,4 +35,9 @@ describe("PseudoLocalization", () => {
       pseudoLocalize("{value, plural, one {# book} other {# books}}")
     ).toEqual("{value, plural, one {# ƀōōķ} other {# ƀōōķś}}")
   })
+
+  it("should pseudolocalize variables", () => {
```

Comment:
I guess this is a typo as it tests that *it shouldn't pseudolocalize variables*

```suggestion
  it("shouldn't pseudolocalize variables", () => {
```